agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/ConfigurationBuilder.java

```diff
@@ -368,6 +371,10 @@ static void overlayEnvVars(Configuration config) throws IOException {
         overlayWithEnvVar(
             APPLICATIONINSIGHTS_PREVIEW_INSTRUMENTATION_RABBITMQ_ENABLED,
             config.preview.instrumentation.rabbitmq.enabled);
+    config.preview.instrumentation.springIntegration.enabled =
+        overlayWithEnvVar(
+            APPLICATIONINSIGHTS_PREVIEW_INSTRUMENTATION_SPRING_INTEGRATION_ENABLED,
```

Comment:
please add a todo for me to add this to Statsbeat feature.

Reply:
👍 will send follow-up PR